queries/snakemake/injections.scm

```diff
@@ -0,0 +1,29 @@
+; inherits: python
+
+(wildcard (constraint) @regex)
+
+;; Injection of bash in shell directives is disabled by default. To enable,
```

Comment:
Why is it disabled?

Reply:
Typically, you'll use wildcards such as `{input}` or `{output}` in the shell directives, which would no longer be highlighted with the injection. If you tend to write complicated scripts in the shell directive, it may be desirable to have proper syntax highlighting in there, but most of the time you're probably better of creating a proper `.sh` file and calling that. I left it in there to show how it's done in case someone wants it, but it could also be removed entirely.

Reply:
We have something similar in `yaml`. Just enable it.

Reply:
I would favor to leave it disabled by default because `{}` has syntactical meaning in bash which is different from its meaning of wildcard interpolation in snakemake and because it will reduce the consistency between the directives in as they're displayed in the rule block vs the shell block.

Reply:
Then I vote for removing them altogether.

Reply:
Then I vote for removing them altogether.

Reply:
[Done](https://github.com/osthomas/nvim-treesitter/commit/321c1e45174ed35663c8a3cef7a92c5c29cbd512#diff-aa5c2fc80b8fec60ad306efda25b19808e8640ff0e37a0cf1bb4beccd490ed3bL5)